source/api/commands/should.md

```diff
@@ -166,9 +166,71 @@ cy
   })
 ```
 
-### Assert explicitly within `.should()`
+{% note warning %}
+Any value returned from `should(cb)` callback function will be ignored. The original subject will passed to the next command automatically.
+
+```
```

Comment:
I have added the warning right next to `should(cb)` even if there is another warning way down the page. Think a little warning is good, even if repeated